src/server/Adaptive.ReactiveTrader.Server.Pricing/PriceSourceAdapters/FixerAdapter.cs

```diff
@@ -0,0 +1,92 @@
+using Newtonsoft.Json.Linq;
+using Serilog;
+using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace Adaptive.ReactiveTrader.Server.Pricing.PriceSourceAdapters
+{
+  /// <summary>
+  /// Scrapes currency pair data from a website called data.fixer.io and returns the currency pairs
+  /// Note https://fixer.io is only free if there are less than a restricted number of API calls per month.
+  /// </summary>
+  public class FixerAdapter : FixerBaseAdapter, IMarketDataAdapter
+  {
+    private const string requestUriString = "https://data.fixer.io/api/latest";
+    private const string sourceName = "Fixer";
+    private const string API_Key = "APIKEY";
+
+    public FixerAdapter() : base(requestUriString)
+    {
+    }
+
+    /// <summary>
+    /// Not currently used because we do not have an API key yet, and the European Central Bank is free!
+    /// </summary>
+    /// <returns></returns>
+    public override async Task<IEnumerable<MarketData>> GetMarketData()
+    {
+      /*
+       The data is returned in this format for each base currency:
```

Comment:
Maybe this could be an XML comment on the method?

Reply:
done